### PR TITLE
Setup tracking GUI for admin

### PR DESCRIPTION
**Pull request by rememberlenny**
https://trello.com/c/nLQACF4D/197-create-gui-to-review-tracking-data-from-ahoy-on-production

Blazer was recently accepted at PR that allows us to customize the authentication. Now that we can authenticate the user's admin status, we can expose Blazer to the production environment.

[FTR, Blazer is awesome](https://github.com/ankane/blazer)

screen shot 2016-03-09 at 11 59 00 pm

### CODE REVIEW

**Comment by pkarman** [comment]
@rememberlenny looks like failing tests.

**Comment by rememberlenny** [comment]
@pkarman got it to go green!

**Comment by rememberlenny** [comment]
@pkarman Thank you!

**Comment by pkarman** [comment]
screen shot 2016-03-10 at 12 34 57 pm

**Comment by pkarman** [comment]
@rememberlenny you might need to add a `config/initializers/blazer.rb` that sets `BLAZER_DATABASE_URL` to `DATABASE_URL` -- since the latter is defined explicitly at cloud.gov.

**Comment by rememberlenny** [comment]
@pkarman Updated!

**Comment by pkarman** [comment]
@rememberlenny not sure that phantomjs change actually helps -- I just manually restarted circleci after another `DeadClient` error.

**Comment by pkarman** [comment]
@rememberlenny I still get `BLAZER_DATABASE_URL required` error here https://c2-dev.18f.gov/blazer/

`c2-dev` is available to test with, if you need.

File: circle.yml [hunk 1]
```diff
@@ -2,5 +2,9 @@ dependencies:
   pre:
     - cp .env.example .env
     - bash ./build-elasticsearch.sh
+    - sudo apt-get update; sudo apt-get install libicu52
```
Comment (pkarman) [inline circle.yml]: I don't think upgrading phantomjs is helping, frankly. I still see DeadClient errors occasionally.

another approach e.g. https://github.com/18F/C2/pull/1043

File: config/routes.rb [hunk 2]
```diff
@@ -63,8 +63,8 @@
   end
 
   mount Peek::Railtie => "/peek"
+  mount Blazer::Engine, at: "blazer"
   if Rails.env.development?
     mount LetterOpenerWeb::Engine => "letter_opener"
-    mount Blazer::Engine, at: "blazer"
```
Comment (pkarman) [inline config/routes.rb]: let's leave this as a dev-only feature, and copy prod db to local env when needed. that keeps blazer security issues to a minimum.

**Pull request closed without merging**